### PR TITLE
Added notebook example for h5 conversion

### PR DESCRIPTION
**Pull request by k-dominik**

### CODE REVIEW

**Review by m-novikov — APPROVED** [review]

File: notebooks/Readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+We use _conda_ to develop Python and recommend it for scientific Python development.
+It is assumed that you have already installed it and are familiar with using a Terminal.
+
+In order to run the notebook type the following in a Terminal/Command Line:
+
+```bash
+$ conda env create -f environment.yml
+# this will produce a lot of output
+
+$ conda activate <created_environment_name>
```
Comment (Tomaz-Vieira) [inline notebooks/Readme.md:21]: won't the name always be h5convert, as defined in the `environment.yml` file?

Comment (k-dominik) [reply notebooks/Readme.md:21]: right :) thx for that one

File: notebooks/Readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Notebook Examples
+
+In this folder you can find [jupyter notebook](https://jupyter.org/) examples to illustrate how to automate certain aspects of, and around ilastik from Python.
+
+The structure in each folder will follow the same scheme:
+
+* notebooks (here)
```
Comment (emilmelnikov) [inline notebooks/Readme.md:7]: Maybe show the directory structure with the output of the `tree` utility?

File: notebooks/Readme.md (hunk 1 above)
Comment (k-dominik) [reply notebooks/Readme.md:21]: ah, wait. This is the top level directory. So readme here should be generic :)

File: notebooks/h5convert/convert_to_h5.ipynb [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+   "cell_type": "markdown",
+   "id": "rocky-season",
+   "metadata": {},
+   "source": [
+    "Note on chunks: This is the most important part to get the performance boost in ilastik.\n",
+    "In hdf5, data can be accessed in blocks (specified via the `chunks` keyword-argument in `create_dataset`.\n",
+    "It follows the same axis order as the image data.\n",
+    "Since ilastik usually processes timepoints and channels independently, we set the chunk size there to `1`.\n",
+    "For 3D data `64` is a sensible choice along the spacial axes.\n",
+    "For 2D data we recommend using `256` alon `x` and `y`."
```
Comment (Tomaz-Vieira) [inline notebooks/h5convert/convert_to_h5.ipynb:87]: *along

File: notebooks/Readme.md (hunk 2 above)
Comment (k-dominik) [reply notebooks/Readme.md:7]: Yeah, okay I will update that - not too happy with the list either, cheers.

**Review by Tomaz-Vieira — APPROVED** [review]

**Pull request merged**